src/app/cart/cart.service.ts

```diff
@@ -60,28 +56,38 @@ export class CartService {
         quantity: 1,
       };
       this.cartData.cartItems = [...this.cartData.cartItems, cartItem];
+      this.updateCartTotalPrice();
+      this.updateCartTotalQuantity();
     }
-    this.updateCartTotalPrice();
-    this.updateCartTotalQuantity();
   }
 
-  increaseQuantity(id: number): void {
+  increaseQuantity(id: number) {
```

Comment:
Add a return type here and split the method into different methods. Read about single responsibility principle